views/js/checkout-component-renderer.js

```diff
@@ -22,7 +22,341 @@
  */
 
 jQuery(document).ready(function () {
-    if (window.ADYEN_CHECKOUT_CONFIG) {
-        window.adyenCheckout = new AdyenCheckout(ADYEN_CHECKOUT_CONFIG);
+    if (!window.ADYEN_CHECKOUT_CONFIG) {
+        return;
+    }
+
+    var placeOrderInProgress = false;
+    var data = {};
+    var placeOrderAllowed;
+    var popupModal;
+
+    var notSupportedComponents = ['paypal'];
+
+    var componentBillingAddress = {};
+    var countryCode = '';
+    var phoneNumber = '';
+    var componentDeliveryAddress = {};
+    var componentPersonalDetails
+    if (typeof prestashop !== 'undefined') {
+        if (selectedInvoiceAddressId in prestashop.customer.addresses) {
+            var invoiceAddress = prestashop.customer.addresses[selectedInvoiceAddressId];
+
+            componentBillingAddress = {
+                city: invoiceAddress.city,
+                country: invoiceAddress.country_iso,
+                houseNumberOrName: invoiceAddress.address2,
+                postalCode: invoiceAddress.postcode,
+                street: invoiceAddress.address1
+            }
+
+            countryCode = invoiceAddress.country_iso;
+            phoneNumber = invoiceAddress.phone ? invoiceAddress.phone : invoiceAddress.phone_mobile;
```

Comment:
Same as above

Reply:
Yes but then the builder won't take them into account